aws/createTemplate.sh

```diff
@@ -50,7 +52,6 @@ GENERATION_INPUT_SOURCE = "${GENRATION_INPUT_SOURCE_DEFAULT}"
 NOTES:
 
 1. You must be in the directory specific to the level
-2. REGION is only relevant for the "product" level
 3. DEPLOYMENT_UNIT must be one of "s3", "cert", "roles", "apigateway" or "waf" for the "account" level
```

Comment:
@roleyfoley , could you please update the numbering below?

Reply:
Fixed up